sources/3.0.0/FC/S-102_Ed_3.0.0_FC.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+<S100FC:S100_FC_FeatureCatalogue xmlns:S100FC = "http://www.iho.int/S100FC/5.2" xmlns:S100Base="http://www.iho.int/S100Base/5.0" xmlns:S100CI="http://www.iho.int/S100CI/5.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:S100CD="http://www.iho.int/S100CD/5.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
Extra spaces:
`xmlns:S100FC = "http://www.iho.int/S100FC/5.2"`

Reply:
done in 42fefa0ba8e1f5e2697bfdc7881735076b134598